orchestrator/Cargo.toml

```diff
@@ -41,3 +41,15 @@ gorc = { path = "./gorc" }
 relayer = { path = "./relayer" }
 gravity_abi_build = { path = "./gravity_abi_build" }
 gravity_abi = { path = "./gravity_abi" }
+
+[features]
+ethermint = [
+    "orchestrator/ethermint",
+    "cosmos_gravity/ethermint",
+    "relayer/ethermint",
+    "gorc/ethermint",
+    "register_delegate_keys/ethermint",
+]
+
+[patch.crates-io]
+deep_space = { git = "https://github.com/iqlusioninc/deep_space/", branch = "master" }
```

Comment:
Unless I'm understanding this incorrectly, isn't this patch directive going to override all the deep_space dependency changes in the individual `Cargo.toml` files?

Reply:
Because `patch.crates-io` only works when the crate `Cargo.toml` depends on a regular dependency and the patched one has the same version number as it.

Reply:
Right, I guess what I don't understand is how this is different from leaving the original dependency in place rather than changing it and then patching back to the original version.

Reply:
> Right, I guess what I don't understand is how this is different from leaving the original dependency in place rather than changing it and then patching back to the original version.

I see, that's because it wasn't `master` branch before our patch get merged into master ;D , and in this way, we only need to change one place to switch the repo and branch, so I think it's good to keep it this way.

Reply:
I guess it does make it cleaner that way.

Is it fine to keep this refactoring @EricBolten or do you prefer to revert the change?

Reply:
I wasn't familiar with this Cargo feature. I think it's fine. It does make things simpler.